### PR TITLE
feat: add seeds.json files

### PR DESCRIPTION
**Pull request by ggutoski**
In preparation for axelarnetwork/axelar-core#1342

### CODE REVIEW

File: resources/testnet/seeds.toml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+[[seed]]
```
Comment (milapsheth) [inline resources/testnet/seeds.toml:1]: Didn't Sergey mention something about not having too many seeds?

File: scripts/host.sh [hunk 2]
```diff
@@ -54,9 +54,12 @@ download_dependencies() {
     msg "copying genesis to configuration directory"
     cp "${shared_directory}/genesis.json" "${config_directory}/genesis.json"
 
-    msg "copying seeds to configuration directory"
+    msg "copying seeds.txt to configuration directory"
     cp "${shared_directory}/seeds.txt" "${config_directory}/seeds.txt"
```
Comment (milapsheth) [inline scripts/host.sh:58]: Isn't this file deleted now (but only for testnet)? We still need to copy this for mainnet.

File: resources/mainnet/seeds.txt [hunk 3]
```diff
@@ -1 +0,0 @@
-3d67d0646cddcc203b41434aceea64ade22ba6fc@k8s-mainnet-axelarco-79b464ee93-f03cb16c57cf7cb2.elb.us-east-2.amazonaws.com:26656,9c9b8eb8cad8648f31d3c9da3d14e825f11a7b52@public-seed.mainnet.axelar.lunanova.tech:26656
```
Comment (milapsheth) [inline resources/mainnet/seeds.txt:1]: Mainnet upgrade will only occur next week. So, this seeds.txt is still needed in the meantime.

**Review by milapsheth — CHANGES_REQUESTED** [review]
We should wait to merge this until just before the upgrade because anyone using the scripts before that won't be able to use it.

File: scripts/host.sh (hunk 2 above)
Comment (milapsheth) [reply scripts/host.sh:58]: Oh yeah, we download the seeds from S3 which is still occurring. `add_seeds` is still being called in `download_dependencies`. Doesn't that clash with the new seeds in the config?

File: resources/testnet/seeds.toml (hunk 1 above)
Comment (ggutoski) [reply resources/testnet/seeds.toml:1]: IIRC that suggestion was an experiment to solve slow testnet syncing.  (It didn't work.)  I see no problem with this many seeds.

File: scripts/host.sh (hunk 2 above)
Comment (ggutoski) [reply scripts/host.sh:58]: I wouldn't say "clash". :P  The new axelar-core version will add seeds found in seeds.toml to whatever seeds it got from conventional sources (eg config.toml).  There might be duplicate seeds from those sources but I don't think it's a problem.

File: resources/mainnet/seeds.txt (hunk 3 above)
Comment (ggutoski) [reply resources/mainnet/seeds.txt:1]: The seeds.txt files in axelarate-community are not used anywhere.  Instead the join scripts download them from S3.  So we need to keep them on S3 (for now at least) but not here.

File: scripts/host.sh (hunk 2 above)
Comment (milapsheth) [reply scripts/host.sh:58]: Don't think we should be duplicating seeds. Scripts should either use `seeds.txt` or `seeds.toml`.

Comment (ggutoski) [reply scripts/host.sh:58]: Sure, but then we need to condition on the version inside the bash script.  Ick.

Good news!  axelar-core de-duplicates all seeds across all sources!
https://github.com/axelarnetwork/axelar-core/blob/a1560807ffdb6d921adad456318b40f3c95bf233/config/seeds.go#L39-L40

So there is no problem with duplicate seeds.

**Comment by ggutoski** [comment]
TODO as discussed:

in the join script:

if > 0.15.0 then use seeds.toml
else parse seeds.toml into a csv and embed into config.toml, similar to what we did for seeds.txt

cli command to parse seeds.toml into csv:

```bash
dasel select -f seeds.toml -w json | jq -r '.seed | map(.address) | join(",")'
```

**Comment by cgorenflo** [comment]
@ggutoski @milapsheth is this still blocked?

**Comment by milapsheth** [comment]
> @ggutoski @milapsheth is this still blocked?

Yeah, we want to simplify the scripts so we'll refactor this during that.

**Pull request closed without merging**